keyless/pepper/example-client-rust/src/main.rs

```diff
@@ -143,13 +139,13 @@ async fn main() {
     };
 
     let dummy_aud = "dummy_aud".to_string();
-    let pepper_request = PepperRequestV1 {
+    let pepper_request = PepperRequest {
         jwt: jwt.clone(),
         epk,
         exp_date_secs: epk_expiry_time_secs,
         uid_key: None,
         epk_blinder: blinder.to_vec(),
-        aud: Some(dummy_aud.clone()),
+        derivation_path: None,
```

Comment:
can we do an `Some(...)` example?

Reply:
I'll add actual tests for this service later